api/src/main/java/io/opentelemetry/correlationcontext/EmptyCorrelationContext.java

```diff
@@ -14,31 +14,31 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.distributedcontext;
+package io.opentelemetry.correlationcontext;
 
 import java.util.Collection;
 import java.util.Collections;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
 /**
- * An immutable implementation of the {@link DistributedContext} that does not contain any entries.
+ * An immutable implementation of the {@link CorrelationContext} that does not contain any entries.
  */
 @Immutable
-public class EmptyDistributedContext implements DistributedContext {
+public class EmptyCorrelationContext implements CorrelationContext {
   private static final Collection<Entry> EMPTY_COLLECTION = Collections.emptyList();
```

Comment:
There's no need for this to be a static field, the `emptyList()` method already returns a singleton instance.

Reply:
Oh, definitely. I don't remember why we had this one as an empty list for a start ;)